src/paths.ts

```diff
@@ -45,6 +45,12 @@ export function convertDebuggerPathToClient(
     } else {
         localPath = path.normalize(serverPath)
     }
+        
+    if (serverIsWindows && !isWindowsUri(localPath)) {
```

Comment:
`localPath` is a path, not a URI, so it's not correct to use `isWindowsUri()` on it

Reply:
Thanks for the input. I am not sure how to fix this and mostly submitted this pull request for the discussion. It does solve the issue on my machine, but if anybody knows a better solution, by all means please help :)